cmip6_preprocessing/preprocessing.py

```diff
@@ -1,11 +1,18 @@
 # Preprocessing for CMIP6 models
 import warnings
 
+import cf_xarray.units
 import numpy as np
 import pandas as pd
+import pint
+import pint_xarray
 import xarray as xr
 
-from cmip6_preprocessing.utils import _maybe_make_list
+from cmip6_preprocessing.utils import _maybe_make_list, cmip6_dataset_id
+
+# global object for units
+_desired_units = {"lev": "m"}
+_unit_overrides = {name: None for name, var in ds.variables.items() if name in ["so"]}
```

Comment:
this will probably have to be
```suggestion
_unit_overrides = {name: None for name in ["so"]}
```

Reply:
Nice